google/cloud/pubsub_v1/gapic/subscriber_client.py

```diff
@@ -525,7 +525,13 @@ def update_subscription(
             >>> client = pubsub_v1.SubscriberClient()
             >>>
             >>> ack_deadline_seconds = 42
-            >>> subscription = {'ack_deadline_seconds': ack_deadline_seconds}
+            >>> subscription_name = 'projects/my-project/subscriptions/my-subscription'
+            >>> topic_name = 'projects/my-project/topics/my-topic'
+            >>> subscription = {
+            ...    'name': subscription_name,
+            ...    'topic': topic_name,
```

Comment:
I checked the `.proto` file and `Subscription.topic` is a required field, too.

Reply:
That's a bit misleading. The "required" field in the Subscription docs refers only to creation time. The "topic" field isn't required for UpdateSubscription. If added WITHOUT adding "topic" to the update_mask, it will have no effect. If added WITH adding to the update_mask, it will result in an error because "topic" is immutable.

So, we should remove the "topic" here.

Reply:
Thanks for clarifying what "required" actually means, will remove the topic field then.